frontend/src/app/components/containers/complaints/outcomes/animal-outcomes/drug-authorization.tsx

```diff
@@ -35,12 +35,12 @@ export const DrugAuthorization: FC<Props> = ({ agency, drugAuthorization, update
   const handleAuthorizedByChange = (input: string | undefined) => {
     setAuthorizedBy(input);
     const newDrugAuth: DrugAuthorizationType = { officer: input ?? "", date: authorizedOn ?? undefined };
-    update(newDrugAuth);
+    update(newDrugAuth, "officer");
   };
 
   const handleAuthorizedOnChange = (input: Date | undefined | null) => {
```

Comment:
is there a need for the null type?

Reply:
If I take the null away it behaves kind of weird when you blank out the date